services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/MetersSpec.groovy

```diff
@@ -358,6 +359,28 @@ class MetersSpec extends BaseSpecification {
         ]
     }
 
+    @Ignore("https://github.com/telstra/open-kilda/issues/2043")
+    def "Try to sync meters for unmetered flow"() {
+        given: "A flow with custom meter rate and burst, that differ from defaults"
+        def randomSwitches = topology.activeSwitches
+        def src = randomSwitches[0]
+        def dst = randomSwitches[1]
+        assumeTrue("Desired switch combination is not available in current topology", src && dst)
+
+        and: "A flow with the 'bandwidth: 0' and 'ignoreBandwidth: true' fields, that differ from defaults"
```

Comment:
Not clear what you mean by saying `that differ from defaults`. I think `A flow with the 'bandwidth: 0' and 'ignoreBandwidth: true'` is enough for describing this step. Or am I missing something?

Reply:
No, you are not. 
I will edit the description
and combine the '_given_' and '_and_' directives